torch/fx/passes/tools_common.py

```diff
@@ -13,7 +14,7 @@
 Names = List[str]
 CALLABLE_NODE_OPS = {"call_module", "call_function", "call_method"}
 
-
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @842974287 FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.